themes/bootstrap3/templates/admin/home.phtml

```diff
@@ -33,12 +33,26 @@
         <tr>
           <th><?=$this->transEsc('Start Time')?>: </th>
           <?php $startTime = $core->xpath('//lst[@name="' . $coreName . '"]/date[@name="startTime"]') ?>
-          <td><?=$this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime(DATE_RFC3339_EXTENDED, (string)array_pop($startTime)))?></td>
+          <td>
+            <?php $startTime = (string)array_pop($startTime); ?>
+            <?php if (!empty($startTime)): ?>
```

Comment:
Might be worth considering converting this to use a ternary for conciseness:

```
            <?=!empty($lastModified)
              ? $this->escapeHtml($this->dateTime()->convertToDisplayDateAndTime(DATE_RFC3339_EXTENDED, $lastModified))
              : $this->transEsc('index_unknown_datetime');
            ?>
```

Reply:
Looks more readable to me, so I've changed it.